integration-tests/src/it-in/snippets/ClassLiteralTests.java

```diff
@@ -92,16 +97,12 @@ static void testMultiDimensionalArrayOfConcreteType() {
         } else {
             putchar('F');
         }
+        putchar('#');
     }
     public static void main(String args[]) {
-        /**
-         * Tests for class literals of primitive type are disabled as
```

Comment:
Just for the record - this comment was incorrect. Before this PR these tests were failing and the real reason for failure was that we are not creating j.l.Class object for primitive types.